app/src/main/java/nl/tudelft/b_b_w/model/TrustValues.java

```diff
@@ -7,7 +7,7 @@
     /**
      * Predefined trust values
      */
-    INITIALIZED(20), REVOKED(0), VERIFIED(50), SUCCESFUL_TRANSACTION(10), FAILED_TRANSACTION(-10);
+    INITIALIZED(10), REVOKED(0), VERIFIED(20), SUCCESFUL_TRANSACTION(10), FAILED_TRANSACTION(-10);
```

Comment:
Because the BlockController will be refactored later, in another pull request. When the blockcontroller will be refactored, the old trust value functions will be removed and so also these dependencies. When the dependencies get removed, these values will be removed too.